SOURCES/CVE-2023-5366-odp-ND-Follow-Open-Flow-spec-converting-from-OF-to-DP.patch

```diff
@@ -0,0 +1,176 @@
+From 29b95c356e0e3514d692b5df442ad5a5d94bbee9 Mon Sep 17 00:00:00 2001
+From: David Morel <david.morel@vates.tech>
+Date: Thu, 22 Feb 2024 10:34:52 +0100
+Subject: [PATCH] odp: ND: Follow Open Flow spec converting from OF to DP
+
```

Comment:
If possible in a few simple sentences, please explain what you had to adapt to make the patch apply.

Reply:
Added notes about the backport.